base/applications/osk/lang/it-IT.rc

```diff
@@ -161,128 +161,129 @@ BEGIN
+    PUSHBUTTON      "fin",SCAN_CODE_81,343,41,15,15
+    PUSHBUTTON      "pg up",SCAN_CODE_85,363,22,15,15,BS_ICON
+    PUSHBUTTON      "pg down",SCAN_CODE_86,363,41,15,15,BS_ICON
+    PUSHBUTTON      "<-",SCAN_CODE_79,323,98,15,15,BS_ICON
+    PUSHBUTTON      "^",SCAN_CODE_83,343,79,15,15,BS_ICON
+    PUSHBUTTON      "v",SCAN_CODE_84,343,98,15,15,BS_ICON
+    PUSHBUTTON      "->",SCAN_CODE_89,363,98,15,15,BS_ICON
+    PUSHBUTTON      "num",SCAN_CODE_90,383,22,15,15
+    PUSHBUTTON      "7",SCAN_CODE_91,383,41,15,15
+    PUSHBUTTON      "4",SCAN_CODE_92,383,60,15,15
+    PUSHBUTTON      "1",SCAN_CODE_93,383,79,15,15
+    PUSHBUTTON      "/",SCAN_CODE_95,403,22,15,15
+    PUSHBUTTON      "8",SCAN_CODE_96,403,41,15,15
+    PUSHBUTTON      "5",SCAN_CODE_97,403,60,15,15
+    PUSHBUTTON      "2",SCAN_CODE_98,403,79,15,15
+    PUSHBUTTON      "0",SCAN_CODE_99,383,98,34,15
+    PUSHBUTTON      "*",SCAN_CODE_100,423,22,15,15
+    PUSHBUTTON      "9",SCAN_CODE_101,423,41,15,15
+    PUSHBUTTON      "6",SCAN_CODE_102,423,60,15,15
+    PUSHBUTTON      "3",SCAN_CODE_103,423,79,15,15
+    PUSHBUTTON      ".",SCAN_CODE_104,423,98,15,15
+    PUSHBUTTON      "-",SCAN_CODE_105,443,22,15,15
+    PUSHBUTTON      "+",SCAN_CODE_106,443,41,15,35
+    PUSHBUTTON      "ent",SCAN_CODE_108,443,78,15,35
+    CTEXT           "Num",IDC_STATIC,383,3,21,8
+    CONTROL         "",IDC_LED_NUM,"Static",SS_CENTER|SS_NOTIFY,391,15,4,3
+    CTEXT           "Caps",IDC_STATIC,408,3,21,8
+    CONTROL         "",IDC_LED_CAPS,"Static",SS_CENTER|SS_NOTIFY,417,15,4,3
+    CTEXT           "Scroll",IDC_STATIC,433,3,21,8
+    CONTROL         "",IDC_LED_SCROLL,"Static",SS_CENTER|SS_NOTIFY,443,15,4,3
 END
 
 IDR_OSK_MENU MENU DISCARDABLE
-    BEGIN
+BEGIN
     POPUP "File"
     BEGIN
-        MENUITEM "&Exit", IDM_EXIT
+        MENUITEM "&Esci", IDM_EXIT
     END
 
-    POPUP "Keyboard"
+    POPUP "Tastiera"
     BEGIN
-        MENUITEM "Enhanced Keyboard", IDM_ENHANCED_KB, CHECKED
-        MENUITEM "Standard Keyboard", IDM_STANDARD_KB
+        MENUITEM "Tastiera Estesa", IDM_ENHANCED_KB, CHECKED
+        MENUITEM "Tastiera Standard", IDM_STANDARD_KB
         MENUITEM SEPARATOR
-        MENUITEM "Regular Layout", IDM_REG_LAYOUT, CHECKED, GRAYED
-        MENUITEM "Block Layout", IDM_BLOCK_LAYOUT, GRAYED
+        MENUITEM "Mappatura Regolare", IDM_REG_LAYOUT, CHECKED, GRAYED
+        MENUITEM "Mappatura a blocchi", IDM_BLOCK_LAYOUT, GRAYED
         MENUITEM SEPARATOR
-        MENUITEM "101 keys", IDM_101_KEYS, CHECKED, GRAYED
-        MENUITEM "102 keys", IDM_102_KEYS, GRAYED
-        MENUITEM "106 keys", IDM_106_KEYS, GRAYED
+        MENUITEM "101 tasti", IDM_101_KEYS, CHECKED, GRAYED
+        MENUITEM "102 tasti", IDM_102_KEYS, GRAYED
+        MENUITEM "106 tasti", IDM_106_KEYS, GRAYED
     END
 
-    POPUP "Settings"
+    POPUP "Impostazioni"
     BEGIN
-        MENUITEM "Always on Top", IDM_ON_TOP, CHECKED, GRAYED
+        MENUITEM "Sempre in cima", IDM_ON_TOP, CHECKED, GRAYED
```

Comment:
Could be better "Sempre in primo piano"? If refferred to a Windows si better. For other element could be no difference into sentence.